lib/src/rpc/dial.dart

```diff
@@ -99,10 +104,28 @@ class DialWebRtcOptions {
   String? signalingAccessToken;
 }
 
+String allowedAddress = '';
+
 /// Connect to a robot at the provided address with the given options
 Future<ClientChannelBase> dial(String address, DialOptions? options, String Function() sessionCallback) async {
   _logger.i('Connecting to address $address');
   final opts = options ?? DialOptions();
+  String? mdnsUri;
+
+  try {
+    mdnsUri = await searchMdns(address);
+    // Let downstream calls know when mdns was used. This is helpful to inform
+    // when determining if we want to use the external auth credentials for the signaling
+    // in cases where the external signaling is the same as the external auth. For mdns
+    // this isn't the case.
+    final dialOptsCopy = opts
+      ..usingMdns = true
+      ..authEntity = address;
+    return _dialWebRtc(mdnsUri, dialOptsCopy, sessionCallback);
+  } catch (e) {
+    _logger.w('error dialing with mDNS; falling back to other methods', error: e);
```

Comment:
I think this should just be `info` or `debug` level, since the user can't do anything about it, it's just to let them know what happened.

Reply:
(Also can we have it be `Error* dialing` for proper capitalization :) )

Reply:
sounds good, had it warning for parody with goUtils, I have no issue with having be debug or info. moving to debug. Thanks!